packages/lumx-react/src/components/select/WithSelectContext.tsx

```diff
@@ -180,6 +181,7 @@ const withSelectContext = (
                 targetUuid={targetUuid}
                 anchorRef={anchorRef}
                 isRequired={isRequired}
+                aria-disabled={isDisabled}
```

Comment:
I might be wrong, but it seems this can never be actually applied ? 

If I understand the code correctly `WithSelectContext` takes a component as a first `SelectElement` argument.
However, both in the `<Select />` and `<SelectMultiple />` components, the components that are set in the `WithSelectContext` function don't have "aria-disabled" as prop and therefore can never set it on the final component.

I have tried to set the demo component as "isDisabled" but can't fin the aria-disabled anywhere ?

![image](https://user-images.githubusercontent.com/7147342/94287167-41892a00-ff56-11ea-952b-4b2eb425ce66.png)
![image](https://user-images.githubusercontent.com/7147342/94287110-333b0e00-ff56-11ea-89c4-a2922f2f50be.png)

Reply:
Yeah I wrongfully assumed the `SelectComponent` would forward the props. Should we put the `aria-disabled` on the `.lumx-select__input-native`?

Reply:
Basically you should put it on the focusable element, since it's the element on which the user will land while navigating with his keyboard :) . In that case, the "__wrapper".

Also if you want to go deeper with this component, you could follow the guidelines shown here : https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html#rps_label

I was gonna list them here, but it might be easier and clearer to just read the documentation and see if it's worth applying them now or later. :) 
(This website is pretty great to see how components should behave a11y wise)